core/arch/arm/tee/entry_fast.c

```diff
@@ -172,6 +172,11 @@ static void tee_entry_vm_created(struct thread_smc_args *args)
 {
 	uint16_t guest_id = args->a1;
 
+	if(guest_id == HYP_CLNT_ID){
```

Comment:
This is better to check in `virt_guest_created()`.

Reply:
Noted.